pandas/tests/extension/base/constructors.py

```diff
@@ -83,3 +83,7 @@ def test_pandas_array_dtype(self, data):
         result = pd.array(data, dtype=np.dtype(object))
         expected = pd.arrays.PandasArray(np.asarray(data, dtype=object))
         self.assert_equal(result, expected)
+
+    def test_construct_empty_dataframe(self, columns, dtype):
```

Comment:
why do you need a columns fixture? just use columns=['a']

this needs a test comparison.